scripts/boot_patch.sh

```diff
@@ -208,8 +208,11 @@ fi
 #################
 
 ui_print "- Repacking boot image"
-./magiskboot repack "$BOOTIMAGE" || abort "! Unable to repack boot image"
-
+if [ `getprop ro.product.brand` == "Xiaomi" -o `getprop ro.product.brand` == "Redmi" ]; then
```

Comment:
Replace this:
```bash
if [ `getprop ro.product.brand` == "Xiaomi" -o `getprop ro.product.brand` == "Redmi" ]; then
```
by this:
```bash
if [ "$(grep_get_prop ro.product.brand)" == "Xiaomi" ] || [ "$(grep_get_prop ro.product.brand)" == "Redmi" ]; then
```
As some Magisk modules may change `ro.product.brand`, this will be more reliable.

Reply:
Instead of hard coding it, we should make it a choice in the app like dm verify:
https://github.com/topjohnwu/Magisk/blob/f42a87b51ae64cf5ed4937e3cd17c7f865f33bc1/app/src/main/res/layout/fragment_install_md2.xml#L93-L100